src/schema/tablecompiler.js

```diff
@@ -205,4 +205,37 @@ TableCompiler.prototype._indexCommand = function (type, tableName, columns) {
   return this.formatter.wrap(indexName);
 };
 
+
+//Default implementation of setNullable. Overwrite on dialect-specific tablecompiler when needed
+//(See postgres/mssql for reference)
+TableCompiler.prototype._setNullableState = function(column, nullable) {
+  let tableName = this.tableName();
+  let columnName = this.formatter.columnize(column);
+  return this.pushQuery({
+    sql: 'SELECT 1',
+    output: () => {
+      return this.client.queryBuilder().from(this.tableNameRaw).columnInfo(column)
+        .then((columnInfo) => {
+          if(isEmpty(columnInfo)) {
+            throw new Error(`.setNullable: Column ${columnName} does not exist in table ${tableName}.`)
+          }
+          let nullableType = nullable ? 'null' : 'not null';
+          let columnType = columnInfo.type + (columnInfo.maxLength ? `(${columnInfo.maxLength})` : '');
+          let defaultValue = (columnInfo.defaultValue !== null && columnInfo.defaultValue !== void 0) ? `default '${columnInfo.defaultValue}'` : '';
+          let sql = `alter table ${tableName} modify column ${columnName} ${columnType} ${nullableType} ${defaultValue}`;
```

Comment:
There is quite a lot copy-paste between this and mssql implementation... could "modify column" string be moved to separate variable and just that variable would be overridden in mssql dialect?